ln_wallet_endpoint.py

```diff
@@ -33,13 +33,16 @@ def lightningd_getinfo():
     rpc = LnRpc()
     return render_template('lightning/lightningd_getinfo.html', info=rpc.get_info())
 
-@ln_wallet.route('/new_address')
+@ln_wallet.route('/new_address/<address_type>', methods=['GET', 'POST'])
```

Comment:
you should probably remove the address type from the url and instead add inside the `if request.method == 'POST':` area `address_type = request.form.get('address_type')`

Reply:
it errors when the address is changed to: "/new_address" and adding "address_type = request.form.get('address_type')"
```
app_1                       | [app_core ERROR] Exception on /ln_wallet/new_address [GET]
app_1                       | Traceback (most recent call last):
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
app_1                       |     response = self.full_dispatch_request()
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
app_1                       |     rv = self.handle_user_exception(e)
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
app_1                       |     return cors_after_request(app.make_response(f(*args, **kwargs)))
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
app_1                       |     rv = self.dispatch_request()
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
app_1                       |     return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
app_1                       |   File "/usr/local/lib/python3.9/site-packages/flask_security/decorators.py", line 509, in decorated_view
app_1                       |     return fn(*args, **kwargs)
app_1                       | TypeError: new_address_ep() missing 1 required positional argument: 'address_type'
```

Reply:
ahhh got it... making  a push later.